lib/pine.coffee

```diff
@@ -11,7 +11,7 @@ class PinejsClientRequest extends PinejsClientCore
 	# @summary Trigger a request to the resin.io API
 	# @private
 	#
-	# @description Makes use of [pinejs-client-js](https://bitbucket.org/rulemotion/pinejs-client-js)
+	# @description Makes use of [pinejs-client-js](https://github.com/resin-io/pinejs-client-js)
 	# You shouldn't make use of this method directly, but through models
 	#
 	# @param {Object} params - request params (same as node-request params)
```

Comment:
Actually I take that back, it should apparently be `params.json = true`, basically match https://github.com/resin-io/pinejs-client-js/blob/master/request.coffee